lib/vsc/install/ci.py

```diff
@@ -177,11 +183,24 @@ def indent(line, level=1):
         indent("// remove untracked files (*.pyc for example)", level=2),
         indent("sh 'git clean -fxd'", level=2),
         indent('}'),
-        indent("stage('test') {"),
     ]
-    lines.extend([indent("sh '%s'" % c, level=2) for c in test_cmds] + [
-        indent('}'),
-    ])
+
+    if vsc_ci_cfg[RUN_SHELLCHECK]:
+        # see https://github.com/koalaman/shellcheck#installing
+        shellcheck_url = 'https://storage.googleapis.com/shellcheck/shellcheck-latest.linux.x86_64.tar.xz'
+        lines.extend([indent("stage ('shellcheck') {"),
+            indent("sh 'curl --silent %s --output - | tar -xJv'" % shellcheck_url, level=2),
+            indent("sh 'cp shellcheck-latest/shellcheck .'", level=2),
+            indent("sh 'rm -r shellcheck-latest'", level=2),
+            indent("sh './shellcheck --version'", level=2),
+            indent("sh './shellcheck bin/*.sh'", level=2),
```

Comment:
we can make the default (to use shellcheck or not) by checking the presence of `.sh` files in bin?

Reply:
I prefer controlling the contents via `vsc-ci.cfg`, there may be cases where we don't actually want `shellcheck` to run on `bin/*.sh` (e.g. an "urgent" fix in a repository that has shell scripts, but hasn't been using `shellcheck` yet).

Also, having a single place that controls what goes in `Jenkinsfile`/`tox.ini` is way better than having various magic ways of enabling/disabling stuff imho...

Reply:
well, you can always turn it off. but i don;t care
we can alwasy change it later